### ISSUE TITLE
Drag & Drop elements, not just click on them

### ISSUE THREAD

**Issue by Mystelven** [lou/multi-select#195]
Hello,

I would like to know if it's possible (and if it is, how ?) to drag and drop elements from one list to another, instead of just clicking on them ?

**Comment by sgtbigman** [comment]
I do not believe this is possible with multi-select, but I can highly recommend [jQuery UI](https://jqueryui.com/)!

**Comment by AbbyJanke** [comment]
Soo seems this is still not an available option, however I did find one that does support drag and drop if still interested:

https://demos.telerik.com/kendo-ui/listbox/drag-and-drop
